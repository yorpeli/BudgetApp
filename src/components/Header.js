import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = ()=>(
    <header>
        <h1>Budget App</h1>
        <NavLink to='/' activeClassName='isActive' exact={true}> Home </NavLink>
        <NavLink to='/create' activeClassName='isActive'> Add </NavLink>
        <NavLink to='/help' activeClassName='isActive'> Help </NavLink>
    </header>
);

export default Header;
