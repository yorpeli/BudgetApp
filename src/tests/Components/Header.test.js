import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';

test ('Header Rendering', ()=>{
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();

    
    //      expect(wrapper.find('h1').text()).toBe('Budget App');
    //     const renderer = new ReactShallowRenderer();
    //     renderer.render(<Header/>);
    //     expect(renderer.getRenderOutput()).toMatchSnapshot();
});
