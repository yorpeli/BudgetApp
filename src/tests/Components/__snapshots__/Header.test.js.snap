// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Rendering 1`] = `
<header>
  <h1>
    Budget App
  </h1>
  <NavLink
    activeClassName="isActive"
    exact={true}
    to="/"
  >
     Home 
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/create"
  >
     Add 
  </NavLink>
  <NavLink
    activeClassName="isActive"
    to="/help"
  >
     Help 
  </NavLink>
</header>
`;
